index.html

```diff
@@ -146,5 +146,31 @@ <h2>Skills</h2>
             </li>
         </ul>
     </section>
+    <section class="contact">
+        <footer class="contact-form">
+        <h2>Contact me</h2>
+        <p>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p>
+        <form action="https://formspree.io/f/xlevdjlz" method="POST">
+            <ul>
+                <li>
+                    <label for="name"></label>
+                    <input type="text" name="user_name" id="name" placeholder="Your Name" maxlength="30" required>
+                </li>
+                <li>
+                    <label for="email"></label>
+                    <input type="email" name="user_email" id="email" placeholder="Enter your email" required>
+                </li>
+                <li>
+                    <label for="message"></label>
+                    <textarea id="message" placeholder="Enter your message here" maxlength="500"></textarea>
```

Comment:
The textarea for the message must also be `required` to prevent a form submission without a message.

Reply:
@KDlamini done! thanks for the great insight